src/User/RegistrationForm.jsx

```diff
@@ -65,8 +65,10 @@ const RegistrationForm = () => {
           />
           <Input
             label="Email Address: "
-            type="text"
+            type="email"
             name="email"
+            pattern="^[a-zA-Z0-9\.]+@[a-zA-Z0-9]+\.[\.A-Za-z]{1,10}"
```

Comment:
I don't think you'll need the anchor elements (^ and $) here but if you use them make sure you use both of them (there's not $ in this pattern)